Examples/Tutorials/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+project(PcapPlusPlusTutorials)
```

Comment:
Maybe we should remove the `project` command?

Reply:
true